crates/wash-cli/src/wit.rs

```diff
@@ -110,3 +128,21 @@ impl FetchArgs {
         Ok("Dependencies fetched".into())
     }
 }
+
+impl PublishArgs {
+    pub async fn run(self) -> anyhow::Result<CommandOutput> {
+        let client = self.common.get_client().await?;
+
+        let (package, version) = client
+            .client()?
+            .publish_release_file(
+                &self.file,
+                PublishOpts {
+                    registry: self.registry,
+                    ..Default::default()
+                },
+            )
+            .await?;
+        Ok(format!("Published {}@{}", package, version).into())
```

Comment:
```suggestion
        Ok(format!("Published {}@{}", package, version).into())
```

Can we have a proper JSON payload for this `CommandOutput` ? Would help with testing it

Reply:
I didn't feel like we needed that for something so simple. exit code 0 == published exit code != 0 is failure. This is purely a wrapper around a function that we are using from an upstream library as well, so most testing is done there (the only thing I might want to test is parsing args, but that felt like overkill for this command)